### ISSUE TITLE
make sure you pass a string to 

### ISSUE THREAD

**Issue by ryanburnette** [therootcompany/auth3000#14]
https://github.com/therootcompany/auth3000/blob/c9b8b50f67272c332ee287e38be2446f69ec7b0c/lib/session.js#L374

Make sure it's a token before you pass it to https://github.com/therootcompany/auth3000/blob/wip2/lib/session.js#L281

or this happens

```
TypeError [ERR_INVALID_ARG_TYPE]: The first argument must be of type string or an instance of Buffer, ArrayBuffer, or Array or an Array-like Object. Received undefined
    at Function.from (buffer.js:333:9)
    at secretToId (/Users/ryan/Development/github.com/edvpbs/api-users/node_modules/auth3000/lib/session.js:281:24)
    at AsyncFunction.<anonymous> (/Users/ryan/Development/github.com/edvpbs/api-users/node_modules/auth3000/lib/session.js:376:14)
    at handleResult (/Users/ryan/Development/github.com/edvpbs/api-users/node_modules/@root/async-router/lib/index.js:85:35)
    at Function.asyncWrapper2 (/Users/ryan/Development/github.com/edvpbs/api-users/node_modules/@root/async-router/lib/index.js:52:20)
    at handleResult (/Users/ryan/Development/github.com/edvpbs/api-users/node_modules/@root/async-router/lib/index.js:85:35)
    at asyncWrapper2 (/Users/ryan/Development/github.com/edvpbs/api-users/node_modules/@root/async-router/lib/index.js:52:20)
    at Layer.handle [as handle_request] (/Users/ryan/Development/github.com/edvpbs/api-users/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/ryan/Development/github.com/edvpbs/api-users/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/ryan/Development/github.com/edvpbs/api-users/node_modules/express/lib/router/route.js:112:3) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

**Comment by ryanburnette** [comment]
I should have documented what I was doing to cause this error.

**Comment by coolaj86** [comment]
> I should have documented what I was doing to cause this error.

agreed

Fixed in ce34d33

**Comment by ryanburnette** [comment]
I'm pretty sure I caused this with an error in my getClaims function where undefined was being passed.
